wannierberri/__system.py

```diff
@@ -284,23 +289,130 @@ def set_symmetry(self,symmetry_gen=[]):
         self.symgroup=Group(symmetry_gen,recip_lattice=self.recip_lattice,real_lattice=self.real_lattice)
 
 
-    #@lazy_property.LazyProperty
-    #def cRvec(self):
-    #    return self.iRvec.dot(self.real_lattice)
+    @lazy_property.LazyProperty
+    def cRvec(self):
+        return self.iRvec.dot(self.real_lattice)
 
     @lazy_property.LazyProperty
     def cRvec_wcc(self):
         """ 
         With self.use_wcc_phase=True it is R+tj-ti. With self.use_wcc_phase=False it is R. [m,n,iRvec] (Cartesian)
         """
-        wannier_centres = self.wannier_centres_cart
-        w_centres = np.array([[j-i for j in wannier_centres] for i in wannier_centres])
+        wannier_centers = self.wannier_centers_cart
+        w_centers = np.array([[j-i for j in wannier_centers] for i in wannier_centers])
         if self.use_wcc_phase:
-            return self.iRvec.dot(self.real_lattice)[None,None,:,:]+ w_centres[:,:,None,:]
+            return self.iRvec.dot(self.real_lattice)[None,None,:,:]+ w_centers[:,:,None,:]
```

Comment:
Remove two lines outside if, and use self.cRvec[None,None, :, :] + self.diff_wcc_cart[:, :, None, ].

Reply:
ok